src/core/src/package_managers/PackageManager.py

```diff
@@ -316,6 +321,47 @@ def set_package_manager_setting(self, setting_key, setting_value=""):
         self.package_manager_settings[setting_key] = setting_value
     # endregion
 
+    # region auto OS updates
+    @abstractmethod
+    def disable_auto_os_update(self):
+        """ Disables auto OS updates on the machine only if they are enabled and logs the default settings the machine comes with """
+        pass
+
+    @abstractmethod
+    def backup_image_default_patch_mode(self):
+        """ Records the default system settings for auto OS updates within patch extension artifacts for future reference.
+        We only log the default system settings a VM comes with, any subsequent updates will not be recorded"""
+        pass
+
+    def image_default_patch_mode_backup_exists(self):
+        """ Checks whether default auto OS update settings have been recorded earlier within patch extension artifacts """
+        self.composite_logger.log_debug("Checking if extension contains a backup for default auto OS updates...")
+        if not os.path.exists(self.image_default_patch_mode_backup_path) or not os.path.isfile(self.image_default_patch_mode_backup_path):
+            self.composite_logger.log_debug("Default system settings for auto OS updates aren't recorded in the extension")
+            return False
+        try:
```

Comment:
Please ensure readability with appropriate new lines. Relates to an old analogy I shared of the readability of book without paragraphs and instead having one long block of text. Apply the comment everything as needed, not just this particular instance.